packages/nimble-components/src/card/styles.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    bodyFont,
+    bodyFontColor,
+    borderColor,
+    borderWidth,
+    mediumPadding,
+    sectionBackgroundColor
+} from '../theme-provider/design-tokens';
+
+// TODO AB#2542845 Replace placeholder style
```

Comment:
Let me know if there's a preferred way to mark placeholders or todo items. This is what I would do in the Skyline repo, but I don't know if it's appropriate here.

Reply:
We don't often link TODOs like this, but if we do, they should link to a nimble issue rather than an azdo work item. If it's short-lived, then you probably don't need a TODO, and we'll just know that it needs to be addressed before closing the issue for the card component. If it's going to stay around for a while, then we often create tech debt issues within nimble to track incomplete work or cleanup that needs to happen.

Reply:
I removed the TODO, since we do plan to update this within the next iteration